src/types/layers.ts

```diff
@@ -0,0 +1,23 @@
+// Here we need to import from the generated code, in order to break
+// a cycle of imports which confuse jest
+// eslint-disable-next-line no-restricted-imports
+import { Layer } from '../oasis-indexer/generated/api'
+import { TFunction } from 'i18next'
+
+export const getLayerNames = (t: TFunction): Record<Layer, string> => ({
+  [Layer.emerald]: t('common.emerald'),
+  [Layer.sapphire]: t('common.sapphire'),
+  [Layer.cipher]: t('common.cipher'),
+  [Layer.consensus]: t('common.consensus'),
+})
+
+interface HasLayer {
+  layer: Layer
+}
+
+export const getFilterForLayer = (layer: Layer) => (item: HasLayer) => item.layer === layer
+
+const preferredOrderOfLayers: Layer[] = [Layer.consensus, Layer.sapphire, Layer.emerald, Layer.cipher]
+
+export const orderByLayer = (itemA: HasLayer, itemB: HasLayer): number =>
+  preferredOrderOfLayers.indexOf(itemA.layer) - preferredOrderOfLayers.indexOf(itemB.layer)
```

Comment:
see https://github.com/oasisprotocol/explorer/pull/426#discussion_r1206026789

Reply:
OK, took the existing layer ordering code in the paratime picker, and factored it out into layer.ts, and using it here, too.

Reply:
Nice